cluster/gce/gci/configure-helper.sh

```diff
@@ -406,25 +410,83 @@ function ensure-local-ssds() {
     return
   fi
   local i=0
-  for ssd in /dev/nvme*; do
+  for ssd in /dev/nvme*n*; do
     if [ -e "${ssd}" ]; then
-      # This workaround to find if the NVMe device is a disk is required because
-      # the existing Google images does not expose NVMe devices in /dev/disk/by-id
-      if [[ `udevadm info --query=property --name=${ssd} | grep DEVTYPE | sed "s/DEVTYPE=//"` == "disk" ]]; then
-        local devicenum=`echo ${ssd} | sed -e 's/\/dev\/nvme0n\([0-9]*\)/\1/'`
-        if [[ "${i}" -lt "${nvmeblocknum}" ]]; then
-          mount-ext "${ssd}" "${devicenum}" "nvme" "block"
-        else
-          mount-ext "${ssd}" "${devicenum}" "nvme" "fs"
-        fi
-        i=$((i+1))
+      # This workaround to find if the NVMe device is a local disk is required
+      # because the existing Google images does not expose NVMe local SSDs in
+      # /dev/disk/by-id
+      if [[ `udevadm info --query=property --name=${ssd} | grep DEVTYPE | sed "s/DEVTYPE=//"` != "disk" ]]; then
+        continue
+      fi
+      if [[ `udevadm info --query=property --name=${ssd} | grep ID_MODEL | sed "s/ID_MODEL=//"` != "nvme_card" ]]; then
+        continue
       fi
+      local devicenum=`echo ${ssd} | sed -e 's/\/dev\/nvme0n\([0-9]*\)/\1/'`
```

Comment:
I think we can't rely on it being `nvme0n`, but we can fix in a followup

Reply:
Yeah let's not change existing behavior (which is what ensure-local-ssds is for, right?)

Reply:
(although nvme[0-9]* would probably do the trick?)

Reply:
assuming that doesn't produce a match group. I need to refresh my `sed` knowledge

Reply:
I reverted changes in the existing function. Let's fix that in a followup